### PR TITLE
Add xml comments to IDocumentBuilder

### PR DESCRIPTION
**Pull request by Shane32**
See 'files changed' for suggested change

### CODE REVIEW

File: src/GraphQL/Execution/IDocumentBuilder.cs [hunk 1]
```diff
@@ -2,8 +2,14 @@
 
 namespace GraphQL.Execution
 {
+    /// <summary>
+    /// Creates a <see cref="Document">Document</see> representing a GraphQL query from a GraphQL request string
```
Comment (sungam3r) [inline src/GraphQL/Execution/IDocumentBuilder.cs]:
```suggestion
    /// Creates a <see cref="Document">Document</see> representing a GraphQL AST from a plain GraphQL query string
```

File: src/GraphQL/Execution/IDocumentBuilder.cs [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
+    /// </summary>
     public interface IDocumentBuilder
     {
+        /// <summary>
+        /// Parse a GraphQL request and return a <see cref="Document">Document</see> representing the request
```
Comment (sungam3r) [inline src/GraphQL/Execution/IDocumentBuilder.cs]:
```suggestion
        /// Parse a GraphQL request and return a <see cref="Document">Document</see> representing the GraphQL request AST
```

**Comment by Shane32** [comment]
What does "AST" stand for?  Perhaps that should be in parentheses?

**Comment by sungam3r** [comment]
AST - Abstract Syntax Tree

**Pull request merged**